vizro-core/examples/visual-vocabulary/pages/time.py

```diff
@@ -160,4 +160,6 @@
         make_code_clipboard_from_py_file("heatmap.py"),
     ],
 )
-pages = [line, column, area, connected_scatter, column_and_line, stepped_line, heatmap]
+
+gantt = gantt_factory("time")
```

Comment:
As @petar-qb already mentioned, no need to use a factory function here :) Just assign the page directly to the gantt chart here given that the gantt chart is only used for the `time` category. We normally only use the factory function if a chart is suitable for several categories. 

See line chart for example code :)